src/node_platform.h

```diff
@@ -84,7 +82,13 @@ class PerIsolatePlatformData :
   static void RunForegroundTask(std::unique_ptr<v8::Task> task);
   static void RunForegroundTask(uv_timer_t* timer);
 
-  int ref_count_ = 1;
+  struct ShutdownCallback {
+    void (*cb)(void*);
+    void* data;
+  };
+  typedef std::vector<ShutdownCallback> ShutdownCbList;
```

Comment:
I remember we usually use `std::list` for these?

Reply:
Uh… is there any reason for that? Unless we need referential integrity or expect a lot of overhead from copying large entries, `std::vector` should be the choice with lower runtime + memory overhead … I think we should switch some of the others over to `std::vector`.

In any case, the expected length of the `ShutdownCbList` here is 1, so `std::list` should definitely not be necessary.